### PR TITLE
Adding documentation for updating k8s.

### PR DESCRIPTION
**Pull request by dlorenc**
cc @aaron-prindle

### CODE REVIEW

**Comment by aaron-prindle** [comment]
LGTM

**Comment by dlorenc** [comment]
Let's hold off on merging for a bit, @aaron-prindle will try these out during the 1.3 update to make sure I haven't missed anything.

File: README.md [hunk 1]
```diff
@@ -237,6 +237,94 @@ export KUBERNETES_CONFORMANCE_TEST=y
...
+ godep restore ./...
+ ```
+
+3. Kubernetes should now be on your GOPATH. Check it out to the right version.
+Make sure to also fetch tags, as Godep relies on these.
+
+ ```shell
+ cd $GOPATH/src/k8s.io/kubernetes
+ git fetch --tags
+ git checkout $DESIREDCOMMIT
```
Comment (jimmidyson) [inline README.md]: Should you do a `godep restore` here too to make sure it picks up any updated/new kubernetes dependencies when you do the minikube `godep save`?

Comment (aaron-prindle) [inline README.md]: Perhaps put a line above git checkout $DESIREDCOMMIT with:
// To list available versions of Kubernetes, use the following command:
$ git tag
v0.10.0
v0.10.1
v0.11.0
....
//Then checkout the desired version using:
git checkout $DESIREDCOMMIT

Comment (dlorenc) [inline README.md]: > Should you do a godep restore here too to make sure it picks up any updated/new kubernetes dependencies when you do the minikube godep save?

Yup! Added that.

Comment (dlorenc) [inline README.md]: > Perhaps put a line above git checkout $DESIREDCOMMIT with:

Added it.

File: README.md [hunk 2]
```diff
@@ -237,6 +237,113 @@ export KUBERNETES_CONFORMANCE_TEST=y
...
+4. Build and test minikube, making any manual changes necessary to build.
+
+5. Update godeps
+
+ ```shell
+ cd $GOPATH/src/k8s.io/minikube
+ rm -rf Godeps/ vendor/
+ godep save ./...
+ ```
+
```
Comment (aaron-prindle) [inline README.md]: Perhaps add a step to verify that the Kubernetes Version was changed:
// Verify that the correct Kubernetes version is being used by minikube:
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"2", GitVersion:"v1.2.4", GitCommit:"3eed1e3be6848b877ff80a93da3785d9034d0a4f", GitTreeState:"clean"}
Server Version: version.Info{Major:"1", Minor:"3+", GitVersion:"v1.3.0-beta.2", GitCommit:"caf9a4d87700ba034a7b39cced19bd5628ca6aa3", GitTreeState:"clean"}

Comment (dlorenc) [inline README.md]: Done. Good idea.

**Comment by jimmidyson** [comment]
1.3.0 has just been tagged - good chance to try this out :smile:

**Comment by dlorenc** [comment]
Nice.

**Comment by luxas** [comment]
I'd prefer this documentation in something like devel, but with a much better name

**Comment by dlorenc** [comment]
+1 on moving most of this stuff to a DEVELOPING.md doc.

**Pull request merged**